tests/protocol-contracts-solana.ts

```diff
@@ -19,6 +19,56 @@ const keyPair = ec.keyFromPrivate('5b81cdf52ba0766983acf8dd0072904733d92afe4dd34
 
 const usdcDecimals = 6;
 
+async function mintSPLToken(conn: anchor.web3.Connection, wallet: anchor.web3.Keypair, mint: anchor.web3.Keypair) {
```

Comment:
I think we should remove all console.log in the test, logging in tests should be minimal by default, should printing passing tests

Reply:
disabled console.log in the test
e905e678536ab80badbd9046137df10c02c15fcc